index.html

```diff
@@ -4,9 +4,9 @@
   <title>Welcome to potionify</title>
   </head>  
 <body>
+  <h1>Welcome to potionify! Enjoy your stay.</h1>
```

Comment:
Could you remove the 'Enjoy your stay' string and make the 'Welcome to potionify ! ' string in red

Reply:
sure! can do

Reply:
great!